src/runtime/recipe/particle.ts

```diff
@@ -238,6 +256,26 @@ export class Particle {
     this.recipe.removeParticle(this);
   }
 
+  getSlotConnectionBySpec(spec) {
+    for (const [name, slotConn] of Object.entries(this._consumedSlotConnections)) {
```

Comment:
Done :) 

It's a little bit different to the code in your comment because consumedSlotConnections is defined as `_consumedSlotConnections: {[index: string]: SlotConnection}` rather than a Map object, but it's essentially the same thing